tests/hardhat/Governance/GovernanceBravo/castVoteTest.ts

```diff
@@ -0,0 +1,309 @@
+import { BigNumber, ContractTransaction, Signer } from "ethers";
+import { ethers, network } from "hardhat";
+import { loadFixture, mine } from "@nomicfoundation/hardhat-network-helpers";
+import { smock, MockContract, FakeContract } from "@defi-wonderland/smock";
+import chai from "chai";
+const { expect } = chai;
+chai.use(smock.matchers);
+
+const { encodeParameters } = require("../../../Utils/BSC");
+
+import {
+  GovernorBravoDelegate,
+  GovernorBravoDelegate__factory,
+  XVSVault,
+  XVSStore,
+  XVS,
+} from "../../../../typechain";
+import { ProposalType } from "../../util/Proposals";
+import { convertToUnit } from "../../../../helpers/utils";
+
+let root: Signer;
```

Comment:
nit: If you wanted to scope these tighter they could go inside the describe block

Reply:
thanks for the suggestion @coreyar . We might refactor the tests later for this purpose :)